Remove dead CLEAN_UP_ISSN cleanup step (#15948)
JabRef/jabref: CHANGELOG.md

### Fixed

> - We fixed a latent crash where the default cleanup preset still referenced the removed `CLEAN_UP_ISSN` step (ISSN normalization now runs via the `NormalizeIssn` field formatter); stored preferences referencing removed cleanup steps are now ignored instead of failing to load. [#15948](https://github.com/JabRef/jabref/pull/15948)

Reviewer: A bit verbosse

```suggestion
- We fixed a crash where the default cleanup preset still referenced the removed `Clean up ISSN` step that was migrated to the `NormalizeIssnFormatter `[#15948](https://github.com/JabRef/jabref/pull/15948)
```

Reviewer 2: Way too much tech babbel. Say only: We fixed a crash when someone tried to do something. User facing change of. For tech stuff GitHub is there.
